docs/nodes/widgets/ui-number-input.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Incorporate ui-number-input in Node-RED Dashboard 2.0 for customizable, user-driven data entry and feedback.
+props:
+    Group: Defines which group of the UI Dashboard this widget will render in.
+    Size: Controls the width of the number input field with respect to the parent group. Maximum value is the width of the group.
+    Icon: Renders a Material Design icon within the Number Input. There is no need to include the "mdi-" prefix.
+    Icon Position: If "Icon" is defined, this property controls which side of the "Label" the icon will render on.
+    Icon Inner Position: If "Icon" is defined, this property controls if icon is render inside or outside the number input box
+    Label: The number shown within the number input field.
```

Comment:
please also mark these as dynamic so that they have the check in the "Dynamic" column:

<img width="759" alt="Screenshot 2024-08-14 at 10 38 16" src="https://github.com/user-attachments/assets/ea5d693d-e39b-49a1-96a9-2523771f836d">

You can do that with:

```
Label:
    description: The number shown within the number input field.
    dynamic: true
````

Reply:
Updated. marked the dynamic props

Reply:
Updated. marked the dynamic props